src/services/exit_order_iterator.py

```diff
@@ -118,6 +141,10 @@ def _operator_delayed_validators(operator_state: NodeOperatorPredictableState) -
 
     @staticmethod
     def _operator_targeted_validators_to_exit(operator_state: NodeOperatorPredictableState) -> int:
+        """
+        If operator has limit, but its validators count is less than limit, then we consider this predicate as 0
+        Because 'targeted limit' shouldn't be used in positive way to sorting not reached to limit operators
```

Comment:
If target limit is higher than predictable operator's validators - it should not have any influence on sorting order